UnityProject/Assets/Scripts/UI/Core/Net/Elements/NetButton.cs

```diff
@@ -23,7 +23,11 @@ public override void ExecuteServer(PlayerInfo subject)
 		public void Awake()
 		{
 			Button = this.GetComponent<Button>();
+			Button.onClick = new Button.ButtonClickedEvent();
 			Button.onClick.AddListener(ExecuteClient);
 		}
+
```

Comment:
what's up with the new empty lines?

Reply:
was debugging stuff and didn't properly cleanup the old functions